RecoTracker/MkFitCore/src/MkFinder.cc

```diff
@@ -1757,12 +1798,12 @@ namespace mkfit {
         if (false)  // (std::isnan(tmp_chi2.At(i, 0, 0)) || tmp_chi2.At(i, 0, 0) > CHI2_BKFIT_CUT)
         {
           // QQQQQ This is wrong -- cur node already increased: !!!!!
-          // QQQQQ HoTNodeArr[ i ][ CurNode[i] ].m_hot.index = -4; // XXXX A new value or use -1 ???
+          // QQQQQ m_HoTNodeArr[ i ][ m_CurNode[i] ].m_hot.index = -4; // XXXX A new value or use -1 ???
 
           // XXXX Should I reduce num hits in TrackCand?
 
-          Par[iC].copySlot(i, Par[iP]);
-          Err[iC].copySlot(i, Err[iP]);
+          m_Par[iC].copySlot(i, m_Par[iP]);
+          m_Err[iC].copySlot(i, m_Err[iP]);
 
           tmp_chi2.At(i, 0, 0) = 0;
         }
```

Comment:
is this `if (false` still needed?
It looks like this was introduced in trackreco/mkFit#275, and was never enabled/used

Reply:
well .... this used to be code that would remove an outlier. I'll remove it and just leave a one line comment that this would be the place to implement hit rejection based on hit chi2.

Reply:
I just removed it all ... if anybody is going to reimplement it they will have to figure out all the details anyway.